README.md

```diff
@@ -0,0 +1,156 @@
+# TuneD: Daemon for monitoring and adaptive tuning of system devices.
+
+(This is TuneD 2.0 with a new code base. If you are looking for the older
+version, please check out branch '1.0' in our Git repository.)
+
+How to use it
+-------------
+
+TuneD is incompatible with ``cpupower`` and ``power-profiles-daemon``. If you
+have these services, uninstall or disable them.
+
+On Fedora, Red Hat Enterprise Linux, and their derivatives: install the ``tuned`` package
+(optionally ``tuned-utils``, ``tuned-utils-systemtap``, and ``tuned-profiles-compat``)::
```

Comment:
Double "::". It's on more places and in GitHub preview it renders as double "::". Is it intended?

Reply:
Hmmm, that must be a Sphinx thing, that's the way we do it for Ceph docs.  I've reverted to single colons here.